components/button/button-icon.js

```diff
@@ -64,7 +64,6 @@ class ButtonIcon extends PropertyRequiredMixin(ThemeMixin(ButtonMixin(VisibleOnA
 					--d2l-button-icon-background-color: transparent;
 					--d2l-button-icon-background-color-hover: var(--d2l-color-gypsum);
 					--d2l-button-icon-border-radius: 0.3rem;
-					--d2l-button-icon-focus-box-shadow: 0 0 0 2px #ffffff, 0 0 0 4px var(--d2l-color-celestine);
```

Comment:
This custom property has been split into two and moved to `button-styles`:
1. `--d2l-button-focus-color`: allows dark mode and translucent buttons to set a different ring colour
2. `--d2l-button-focus-offset`: allows search inputs to shrink the offset to `1px` and translucent buttons to invert it and make the focus ring on the inside